packages/frontend/editor-ui/src/components/RunDataJsonActions.vue

```diff
@@ -39,9 +39,13 @@ const props = withDefaults(
 const ndvStore = useNDVStore();
 const workflowsStore = useWorkflowsStore();
 
+const pipWindow = inject(PiPWindowSymbol, ref<Window | undefined>());
```

Comment:
Maybe we can inject PiP window in useClipboard itself so that we don't have to always consider if the component can be used in PiP or not

Reply:
Because I can't think of a situation where PiP window is provided but should not pass it down to vueuse/core's useClipboard